cypress/integration/tests/harvest-register.spec.js

```diff
@@ -17,6 +17,14 @@ const fishPenAfterHarvestValueForM2 = 64000;
 const penM2Comment = "Comment for pen M2";
 const successfulToasterPopupMessage = 'Saved successfully';
 const reportDate = Cypress.moment().format("DD/MM/YYYY");
+const updatedFishHarvestedValueForPenM1 = 70000;
```

Comment:
`const` is a signal that the identifier won’t be reassigned.
`let` is a signal that the variable may be reassigned. 
https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75

are you sure it should be const?